output/network.go

```diff
@@ -33,16 +31,5 @@ type Network struct {
 
 // HasChanged diff's the two items and returns true if there have been any changes
```

Comment:
@purehyperbole could you update the comment so its clear why a network can't change please